### PR TITLE
chore: various fixes after merging

### PR DESCRIPTION
**Pull request by anniehu4**
### Summary:
this is what I get for force pushing 🥲 

### Test Plan:
- `yarn start` actually loads storybook correctly (locally)
- `yarn types`, `yarn lint`, `yarn prettier` all pass
- hopefully chromatic shows no changes (or no unexpected changes)

### CODE REVIEW

File: .storybook/main.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ module.exports = {
   ],
   webpackFinal: async (config) => {
```
Comment (jinlee93) [inline .storybook/main.js:36]: https://61313967cde49b003ae2a860-tymygfwjmc.chromatic.com/?path=/story/atoms-icons-icongrid--default Icons seem to load, so not exactly sure what that `icon-component-loader` file was for, might be remnants of old storybook versions, like the separate webpack config

**Comment by anniehu4** [comment]
alas ... looks like all the stories are new now -- i'm doing a quick scan and it looks ok tho

**Comment by jinlee93** [comment]
So in main, I moved `Icon Grid` as a story under `Icon`, so as a result of the merge there are 2 Icon Grid stories now. Can you remove one of them? 
image

**Review by dierat — APPROVED** [review]

**Comment by anniehu4** [comment]
> So in main, I moved Icon Grid as a story under Icon, so as a result of the merge there are 2 Icon Grid stories now. Can you remove one of them?

ahh shoot, yes

**Review by jinlee93 — APPROVED** [review]

**Pull request merged**